src/NUnitFramework/nunit.framework.legacy/LegacyExtensions.Comparisons.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+
+// ReSharper disable once CheckNamespace
+namespace NUnit.Framework
+{
+    /// <summary>
+    /// C#14 static extension methods for comparison assertions (Greater, Less, etc.)
+    /// </summary>
+    public static class LegacyAssertComparisonExtensions
+    {
+        extension(Assert)
+        {
+            #region Greater
+
+            public static void Greater(int arg1, int arg2, string? message = null, params object[] args) => Legacy.ClassicAssert.Greater(arg1, arg2, message ?? string.Empty, args);
```

Comment:
3.14 doesn't have nullable _message_. It has two overloads, one with and one without _message_ parameter.
Should we not do the same here?